Common/Systems/BossTrackingSystems/BossTracker.cs

```diff
@@ -210,8 +207,11 @@ public override void LoadWorldData(TagCompound tag)
 		if (tag.ContainsKey("DownedFlags"))
 		{
 			BitsByte oldMask = tag.GetByte("DownedFlags");
-			if (oldMask[0]) { EventTracker.CompleteEvent(EventFlags.DefeatedEaterOfWorlds); }
-			if (oldMask[1]) { EventTracker.CompleteEvent(EventFlags.DefeatedEaterOfWorlds); }
+
+			// These calls are counted as "fromSync" since they don't need to sync - incoming players will have the result synced for them, and
+			// at the time this is loaded players can't be joined to the server already
+			if (oldMask[0]) { EventTracker.CompleteEvent(EventFlags.DefeatedEaterOfWorlds, null, true); }
+			if (oldMask[1]) { EventTracker.CompleteEvent(EventFlags.DefeatedEaterOfWorlds, null, true); }
```

Comment:
Fixes a mistake of my own.
```suggestion
			if (oldMask[0]) { EventTracker.CompleteEvent(EventFlags.DefeatedEaterOfWorlds, fromSync: true); }
			if (oldMask[1]) { EventTracker.CompleteEvent(EventFlags.DefeatedBrainOfCthulhu, fromSync: true); }
```